Translations/uk.lproj/Credits.strings

```diff
@@ -12,4 +12,4 @@
 "_SOURCE_" = "Вихідний код Keka 1.0 не буде публічним через деякі правові питання. Потрібна юридична підтримка, якщо ви можете допомогти проекту, будь ласка, зв’яжіться за адресою info@keka.io або на сторінці проекту на офіційному сайті Keka. Будь-яка допомога вітається.";
 "_WARRANTY_" = "THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS \"AS IS\" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.";
 "_THANKSTOALL_" = "Дякуємо всім донорам і всім, хто купив Keka з App Store.<br />Дякуємо за всі відгуки та випадкові внески.<br />Дякуємо Ігорю Павлову за такий чудовий та відритий застосунок, і перенесення до POSIX.<br /><br />Ви всі тримаєте проект Кека живим і даєте можливість його розвивати.";
-"_APPEARDONATION_" = "If you donated and want to appear here, just <a href=\"mailto:info@keka.io?subject=Keka donation credits\">let me know</a> :)";
+"_APPEARDONATION_" = "Якщо ви пожертвували і хочете з'явитися тут, просто <a href=\"mailto:info@keka.io?subject=Keka реквізити для пожертв\">дайте мені знати</a> :)";
```

Comment:
Do we need to translate the email subject? Keeping it English makes more sense for @aonez

Reply:
I've never received a mail for that one anyway 🤷🏻‍♂️